src/Confluent.Kafka/ISerializingProducer.cs

```diff
@@ -49,42 +49,5 @@ public interface ISerializingProducer<TKey, TValue>
         /// <include file='include_docs_producer.xml' path='API/Member[@name="ProduceAsync_string_Partition_TKey_TValue_Timestamp_IEnumerable"]/*' />
         /// <include file='include_docs_producer.xml' path='API/Member[@name="Produce_Action"]/*' />
         void Produce(Action<Message<TKey, TValue>> deliveryHandler, string topic, Partition partition, TKey key, TValue val, Timestamp timestamp, IEnumerable<Header> headers);
-
-#region obsolete produce methods
-
-        /// <include file='include_docs_producer.xml' path='API/Member[@name="ProduceAsync_Obsolete"]/*' />
-        [Obsolete("Variants of ProduceAsync that include a blockIfQueueFull parameter are depreciated - use the " + Producer.BlockIfQueueFullPropertyName + " configuration property instead.")]
-        Task<Message<TKey, TValue>> ProduceAsync(string topic, TKey key, TValue val, int partition, bool blockIfQueueFull);

```

Comment:
rather than continue with this mess past 1.0.x, I think we should support 0.11.x for longer.